tests/pr_test.go

```diff
@@ -85,7 +85,24 @@ func TestRunBankingSolutions(t *testing.T) {
 			},
 		})
 
-		output, err := options.RunTest()
+		// Ignore updates
+		options.IgnoreUpdates = testhelper.Exemptions{
+			List: []string{
+				"ibm_cd_tekton_pipeline_property.watsonx_assistant_integration_id_pipeline_property_cd",
+				"ibm_cd_tekton_pipeline_property.watsonx_assistant_integration_id_pipeline_property_ci",
+			},
+		}
+
+		// Ignore destroys
+		options.IgnoreDestroys = testhelper.Exemptions{
+			List: []string{
+				"null_resource.discovery_file_upload",
+				"null_resource.assistant_project_creation",
```

Comment:
why are you ignoring these destroys? It means the test will never test destroying these, and you don't want that since these are created by the DA itself.